package.json

```
@@ -62,6 +62,6 @@
     "sinon-chai": "^3.1.0"
   },
   "engines": {
-    "node": "6.* || >= 8.*"
+    "node": "8.* || >= 10.*"
```

We do not want to support Node 11 (it is unsupported by Node itself at this point):

```suggestion
    "node": "8.* || 10.* || >= 12.*"
```

---

Should I add the node 12 to the CI as well?

---

Yeah, probably